rascaline-c-api/CMakeLists.txt

```diff
@@ -86,30 +88,6 @@ if (NOT CARGO_EXE)
     )
 endif()
 
-execute_process(
-    COMMAND ${CARGO_EXE} "--version"
-    RESULT_VARIABLE CARGO_STATUS
-    OUTPUT_VARIABLE CARGO_VERSION
-)
-
-if(CARGO_STATUS AND NOT CARGO_STATUS EQUAL 0)
-    message(FATAL_ERROR
-        "could not run cargo, please make sure the Rust compiler is installed \
-        (see https://www.rust-lang.org/tools/install)"
-    )
-endif()
-
-set(REQUIRED_RUST_VERSION "1.53.0")
-string(REGEX REPLACE "cargo ([0-9]+\\.[0-9]+\\.[0-9]+).*" "\\1" CARGO_VERSION ${CARGO_VERSION})
-if (${CARGO_VERSION} VERSION_LESS ${REQUIRED_RUST_VERSION})
-    message(FATAL_ERROR
-        "your Rust installation is too old (you have version ${CARGO_VERSION}), \
-        at least ${REQUIRED_RUST_VERSION} is required"
```

Comment:
We no longer need to check for the rust version, since cargo does it for us now

Reply:
I like!